src/components/Toast/LoadingToast.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from "react"
+
+import Spinner from "@/components/Spinner"
+import { Toast, TransactionToastComponentProps } from "@/components/Toast"
+
+export const LoadingToast: FC<TransactionToastComponentProps> = ({
+  message = "Loading...",
```

Comment:
This should be a required prop. I don't know of any case where we'd fire off a toast without a message.